tests/uefistored/test_auth_var.py

```diff
@@ -62,3 +65,53 @@ def test_auth_variable(imported_vm):
 
     # Set the variable with new data, signed by a different cert
     set_and_assert_var(vm, Certificate(), b'this should fail', should_pass=False)
+
+    vm.shutdown(verify=True)
+
+
+def test_db_append(linux_uefi_vm):
+    """Pass if appending the DB succeeds. Otherwise, fail."""
+    vm = linux_uefi_vm
+    PK, KEK, db, db2 = EFIAuth("PK"), EFIAuth("KEK"), EFIAuth("db"), Certificate("db")
+    PK.sign_auth(PK)
+    PK.sign_auth(KEK)
+    KEK.sign_auth(db)
+
+    vm.install_uefi_certs([PK, KEK, db])
+    vm.start()
+    vm.wait_for_os_booted()
+
+    vm_kek_key = vm.ssh(['mktemp'])
+    vm.scp(KEK.cert.key, vm_kek_key)
+
+    vm_db_cert = vm.ssh(['mktemp'])
+    vm.scp(db2.pub, vm_db_cert)
+
+    _, old = vm.get_efi_var(db.name, db.guid)
+
+    vm.ssh(
+        [
+            "chattr",
+            "-i",
+            "/sys/firmware/efi/efivars/db-d719b2cb-3d3a-4596-a3bc-dad00e67656f"
+        ]
+    )
+
+    vm.ssh(
+        [
+            "efi-updatevar",
+            "-k",
+            vm_kek_key,
+            "-c",
+            vm_db_cert,
+            "-a",
+            "db",
+        ]
```

Comment:
I don't find this much readable for ssh commands, but I won't argue :)

Reply:
Why not this instead:
```python
vm.ssh([
    "efi-updatevar",
    "-k",
    vm_kek_key,
    "-c",
    vm_db_cert,
    "db"
])
```

A single level of indentation is quite sufficient. :)

Reply:
Or one line: `vm.ssh(["efi-updatevar", "-k", vm_kek_key, "-c", vm_db_cert, "db"])`.
The max char limit is not reached.